libs/designsystem/src/lib/components/page/page.component.spec.ts

```diff
@@ -146,22 +146,13 @@ describe('PageComponent', () => {
     it('should trigger onEnter when navigating to url', async () => {
       const enterEmitSpy = spyOn(spectator.component.enter, 'emit');
 
-      await zone.run(() => router.navigate(['']));
-      spectator.detectChanges();
-
-      expect(enterEmitSpy).toHaveBeenCalledTimes(1);
-    });
-    it('should trigger onEnter once when navigating to same twice url', async () => {
-      const enterEmitSpy = spyOn(spectator.component.enter, 'emit');
-
-      await zone.run(() => router.navigate(['']));
       spectator.detectChanges();
```

Comment:
Please add comment as to why `spectator.detectChanges` is needed - or put it in a beforeEach for all nav tests...